modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/internal/util/GraphQLNamingUtil.java

```diff
@@ -22,15 +24,29 @@ public static String getGraphQLMutationName(String methodName) {
 	}
 
 	public static String getGraphQLPropertyName(
-		String methodName, String returnType, List<String> methodNames) {
+		JavaMethodSignature javaMethodSignature, List<String> methodNames) {
 
-		if (!methodName.equals("getSite") &&
+		String methodName = javaMethodSignature.getMethodName();
+		String parentSchemaName = javaMethodSignature.getParentSchemaName();
+		String returnType = javaMethodSignature.getReturnType();
+
+		if (!methodName.endsWith("SitesPage") &&
```

Comment:
I think prior your change, this was sorted in a different way. contains way before endswith